apps/web/src/ee/clerk/pages/ManageAccountPage.styles.ts

```diff
@@ -0,0 +1,92 @@
+import { css } from '@novu/novui/css';
+
+export const clerkComponentAppearance = {
+  elements: {
+    navbar: { display: 'none' },
+    navbarMobileMenuRow: { display: 'none !important' },
+    cardBox: {
+      display: 'block',
+      width: 'full',
+      height: 'full',
+      boxShadow: 'none',
+    },
+  },
+};
+
+export const modalStyles = {
+  modal: css({
+    borderRadius: '150 !important',
+    backgroundColor: 'surface.panel !important',
+    padding: '0 !important',
+    width: '[90%] !important',
+    height: 'full !important',
+  }),
+  body: css({ height: 'full !important' }),
+  header: css({ marginBottom: '0 !important' }),
+  close: css({
+    position: 'fixed !important',
+    right: '[12px] !important',
+    top: '[10px] !important',
+    zIndex: '[2] !important',
+  }),
+};
+
+export const titleTab = css({
+  fontWeight: 'strong',
+  color: 'typography.text.main !important',
+  fontSize: 'x-large !important',
+  lineHeight: '175 !important',
+  opacity: '100 !important',
+  margin: '0 !important',
+  border: '0 !important',
+  padding: '150 !important',
+  '&:hover': {
+    color: 'typography.text.main !important',
+    backgroundColor: 'unset !important',
+    borderColor: 'surface.panel !important',
+    cursor: 'unset !important',
+  },
+});
+
+export const billingTitle = css({
+  fontSize: '125',
+  letterSpacing: '0',
+  lineHeight: '175',
+});
+
+export const normalTabStyle = css({
+  fontWeight: 'strong',
+  fontSize: '88 !important',
+  color: 'typography.text.secondary !important',
+  padding: '[8px 12px] !important',
+  margin: '[4px]',
```

Comment:
```suggestion
  py: '50 !important',
  px: '75 !important',
  margin: '25',
```
Our spacing / dimension values work based on 16px / 1 rem as 100%, so the token for that is '100'. If you need 4px which is 0.25rem (or 25% of 1rem), then the value is `'25'`

Reply:
Thank you, I was trying to conclude what the ratio is, this is very helpful :)